lib/internal/quic/core.js

```diff
@@ -1931,11 +1969,7 @@ class QuicSession extends EventEmitter {
       return;
     state.destroyed = true;
 
-    // Destroy any pending streams immediately. These
-    // are streams that have been created but have not
-    // yet been assigned an internal handle.
-    for (const stream of state.pendingStreams)
-      stream.destroy(error);
+    state.idleTimeout = Boolean(this[kInternalState].state?.idleTimeout);
```

Comment:
This being a boolean surprised me. What does it indicate?

Reply:
It indicates that the `QuicSession` was destroyed due to the idle timeout expiring.